SPECS/rubygem-bigdecimal/rubygem-bigdecimal.spec

```diff
@@ -1,54 +0,0 @@
-%global debug_package %{nil}
-%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
-%global gem_name bigdecimal
-Summary:        BigDecimal provides arbitrary-precision floating point decimal arithmetic
-Name:           rubygem-bigdecimal
-Version:        3.1.2
```

Comment:
It looks like the version here is higher than the default one from `ruby.spec`, so we shouldn't remove it. We still want to keep track of any CVEs for this version, since we already published it and it's going to be preferred to the `ruby` package. I'd keep this separate spec for now until `ruby` moves forward ahead, that we can drop it.

Reply:
Done